### DISCUSSION TITLE
Is there a way to exclude certain types from the search api

### DISCUSSION THREAD

**Discussion by nathan-at-tes** [osm-search/Nominatim#2898] [Q&A]
**Is your feature request related to a problem? Please describe.**

This may exist, I've looked, but can't find it.

Our problem is we want to search locations, but don't want a high level of granularity. So searching countries, cities etc is fine, but we don't want to find airports in our results.

**Describe the solution you'd like**

We would like to be able to only search for countries, cities, states, etc.

**Comment by mtmail** [comment]
For reverse search there's the `zoom` parameter https://nominatim.org/release-docs/latest/api/Reverse/#result-limitation You can test it on https://nominatim.openstreetmap.org/ui/reverse.html

For forward search there's no such parameter. You'd need to import less data into your database. https://nominatim.org/release-docs/latest/admin/Import/#filtering-imported-data
